File: Algoritmos/1º Período/java/L1e2_SomadosQuadrados.java
/*
 * Autor: Matheus Scaketti
 * Data: 16/03/2016
 * 
 * Lista: 1
 * Exercício: 2
 * 
 * Descrição:
 * O exercício pede que seja inserido 3 valores para a, b e c e calcule
 * a soma dos quadrados dos valores inseridos
*/

import java.util.Scanner;

public class L1e2_SomadosQuadrados {

	public static void main(String[] args) {
		
		float a, b, c, result;
		
		System.out.println("Insira os valores para A, B e C: ");
		
		Scanner leitor = new Scanner(System.in);
		
		a = leitor.nextFloat();
		b = leitor.nextFloat();
		c = leitor.nextFloat();
		leitor.close();
		
		//calcula a soma dos quadrados dos números
		result = (a * a) + (b * b) + (c * c);
		
		System.out.println("A soma dos quadrados dos números é: " + result);
	}

}

File: Algoritmos/1º Período/C/L1e2_SomaDosQuadrados.c
/*
 * Autor: Othon Alberto
 * Data: 18/03/2016
 *
 * Lista: 1
 * Exercício: 2
 *
 * Descrição:
 * O exercicio pede para que seja inserido 3 valores e que retorne a soma dos quadrados desses número
 */

#include <stdio.h>

int main(){
    
    float a, b, c, soma;
    
    printf("Digite o primeiro valor: ");
    scanf("%f", &a);
    
    printf("Digite o segundo valor: ");
    scanf("%f", &b);
    
    printf("Digite o terceiro valor: ");
    scanf("%f", &c);
    
    //calcula a soma dos quadrados
    soma = (a*a) + (b*b) + (c*c);
    
    printf("O valor da soma dos quadrados dos números inseridos é: %.2f\n", soma);
    
    
}